### PR TITLE
feat: Cleanup cluster entities

### PR DESCRIPTION
**Pull request by adityabhatia**

### CODE REVIEW

File: pkg/scheduler/service/cleaner.go [hunk 1]
```diff
@@ -53,18 +53,30 @@ func (c *cleaner) Run(ctx context.Context, transition *ClusterStatusTransition,
...
+		c.logger.Infof("%s Cleaner will remove unnecessary reconciliations", CleanerPrefix)
 		c.purgeReconciliationsNew(transition, config)
 	} else {
-		c.logger.Infof("%s Cleaner will remove entities older than %s", CleanerPrefix, config.PurgeEntitiesOlderThan.String())
+		c.logger.Infof("%s Cleaner will remove reconciliations older than %s", CleanerPrefix, config.PurgeEntitiesOlderThan.String())
 		c.purgeReconciliationsOld(transition, config)
 	}
 
+	// delete cluster entities
+	clusterInventoryCleanupDays := 20 // days default
```
Comment (Tomasz-Smelcerz-SAP) [inline pkg/scheduler/service/cleaner.go:69]: I think that the default should not be set here, but should instead come from configuration/flags definition

**Review by Tomasz-Smelcerz-SAP — APPROVED** [review]

File: pkg/scheduler/service/cleaner.go (hunk 1 above)
Comment (adityabhatia) [reply pkg/scheduler/service/cleaner.go:69]: Makes sense, let's take a decision on this in a follow-up

**Pull request merged**